guides/common/modules/proc_upgrading-rhel7-to-rhel8.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Upgrading-Project-from-RHEL7-to-RHEL8_{context}"]
+= Upgrading {Project} From {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+
+.Prerequisites
+* {Project} {ProjectVersion} is supported on both {RHEL} 7 and {RHEL} 8.
+* Subsequent minor versions of {Project} 6 will potentially be supported on {RHEL} 8 only.
+* If you have {Project} {ProjectVersion} running on {RHEL} 7, you must migrate to {RHEL} 8 to stay up-to-date on {Project} {ProjectVersion} minor releases.
+* {Project} {ProjectVersion} running on {RHEL} 7.
+* An offline backup created by `satellite-maintain`.
+This can be an older full backup with incremental backups leading to the point from where you want to restore.
+	*** Not supported by `satellite-clone`.
+* An identical machine that can be configured with the same hostname and IP address as the source.
+
+.Fresh Install Migration
+. The fresh install migration method allows you to create a {RHEL} 8 system with identical specifications and the same hostname as the {RHEL} 7 system.
+. Additionally, you can choose how much of the {RHEL} 7 system to include in the backup, allowing you to selectively import only relevant parts of the {RHEL} 7 system to {RHEL} 8.
+. This method requires a considerable amount of system downtime while the system backup is restored on the {RHEL} 8 system.
+. To reduce the amount of downtime, you can continue to use the {RHEL} 7 system on the target machine while the backup is being restored.
+. Any new changes you make during this time are not included in the backup on the {RHEL} 8 system on the source machine.
+. Storage space is required for the backup and access to a second machine to host the new {RHEL} 8 system.
+
+.In-Place Migration Using LEAPP
+. The in-place migration method requires less downtime (typically several hours) than the fresh install method.
```

Comment:
Please do not use parenthesis. Reword to include in a sentence or write as a sentence on its own.